tools/CMakeLists.txt

```diff
@@ -361,7 +370,9 @@ function(pico_sign_binary TARGET)
 endfunction()
 
 # pico_hash_binary(TARGET)
-# Hash the target binary. This sets PICOTOOL_HASH_OUTPUT to true.
+# Hash the target binary.
+#
```

Comment:
just picked this at random, but this sets the target property PICOTOOL_HASH_OUTPUT